apps/app/src/views/routes/Prescription.tsx

```diff
@@ -68,6 +69,7 @@ export const CANCEL_PRESCRIPTION = gql`
 export const Prescription = () => {
   const params = useParams();
   const id = params.prescriptionId;
+  const hasPermission = usePermissions(['write:prescription']);
```

Comment:
Does write:prescription also require auth check?

Reply:
not sure I follow, these permissions are coming available after authentication